### PR TITLE
webpack output naming

### PR DESCRIPTION
**Pull request by rwd**
Primary changes, to webpack output config:
* Name the library "EuropeanaMediaPlayer" and export default so that when included in an HTML `script` element it may be instantiated like `new EuropeanaMediaPlayer` (instead of `new europeanamediaplayer.default`
* Name the output directory "dist", and the file "europeana-media-player.min.js", in line with common convention

Additional housekeeping changes:
* Fix the out-of-sync version number in package-lock.json
* Remove `files` from package.json to prevent installing unneeded files in downstream projects, including heavy binary media files
* Delete the source map from the code repository
* Lint the webpack config file

### CODE REVIEW

**Comment by rwd** [comment]
@andyjmaclean [by default](https://docs.npmjs.com/files/package.json#files) npm includes the file defined in the `main` property. I would therefore expect the derived asset, i.e. the bundled JS, to be included. Have you experienced otherwise?

**Comment by andyjmaclean** [comment]
> @andyjmaclean [by default](https://docs.npmjs.com/files/package.json#files) npm includes the file defined in the `main` property. I would therefore expect the derived asset, i.e. the bundled JS, to be included. Have you experienced otherwise?

Version 0.5.21 never picked it up, and that test did not reference the asset in .gitignore

See: https://github.com/europeana/media-player/commit/e4d018c0bababfa5952fa84fdb1958f7b28d1b49

**Comment by rwd** [comment]
`npm publish --dry-run` on this branch indicates that the main bundle output file is included, but other files in dist/ will not be (now, with commit https://github.com/europeana/media-player/pull/74/commits/e4288d913ae06772fbb472678d052778a5662dbc)

File: webpack.config.js [hunk 1]
```diff
@@ -134,26 +132,24 @@ const icoRule = function() {
...
 const output = function() {
-  let output = {
-    publicPath: '/',
-    path: path.resolve(__dirname, './lib'),
-    filename: outputFile,
-    library: libraryName,
+  return {
+    filename: 'europeana-media-player.min.js',
+    library: 'EuropeanaMediaPlayer',
+    libraryExport: 'default',
```
Comment (pietervanleeuwen) [inline webpack.config.js:138]: This also requires an update in the [README - Embed player](https://github.com/europeana/media-player#embed-player) on how to require the player (leave out the default)

**Pull request merged**